### PR TITLE
fix: remove travis and add coverage

### PR DESCRIPTION
**Pull request by vishdha**
**Decription**

Removed travis .yml

### CODE REVIEW

File: .github/workflows/server-tests.yml [hunk 1]
```diff
@@ -70,6 +70,6 @@ jobs:
         run: bash ${GITHUB_WORKSPACE}/.github/helper/install.sh
 
       - name: Run Tests
-        run: cd ~/frappe-bench/ && bench --site test_site run-tests --app erpnext
+        run: cd ~/frappe-bench/ && bench --site test_site run-tests --app erpnext --coverage
```
Comment (sahil28297) [inline .github/workflows/server-tests.yml]:
```suggestion
        run: cd ~/frappe-bench/ && bench --site test_site run-tests --app erpnext
```
I'll add coverage command separately.

**Review by sahil28297 — CHANGES_REQUESTED** [review]

**Review by sahil28297 — CHANGES_REQUESTED** [review]
delete the .travis folder as well

**Comment by neilLasrado** [comment]
@vishdha whats going on with this PR?

**Comment by neilLasrado** [comment]
@vishdha @sahil28297 what's going on with this?

**Comment by sahil28297** [comment]
Closing this, refer https://github.com/Bloomstack/erpnext/pull/1815

**Pull request closed without merging**